server/src/cli.rs

```diff
@@ -26,19 +25,18 @@ pub fn parse() -> Arguments {
 #[clap( author="The Ankaios team",
         version=env!("CARGO_PKG_VERSION"),
         about="Ankaios - your friendly automotive workload orchestrator.\nWhat can the server do for you?")]
+// default values for the server are set in server-config.rs
 pub struct Arguments {
     #[clap(short = 'c', long = "startup-config")]
     /// The path to the startup config yaml.
```

Comment:
We had a small discussion about the configs in the desired state and noticed that we have too many "configs" in Ankaios. It would be better to pinpoint the terms and for that we should also correct the name of the startup-config:
```suggestion
    #[clap(short = 'c', long = "startup-manifest", alias = "startup-config")]
    /// The path to the startup manifest yaml.
```
Which this solution we still retain backwards compatibility and can introduce a new name. Still, please update all internal usage from "startup-config" to "startup-manifest". Thx!